### DISCUSSION TITLE
未来是否有支持websocket的计划？

### DISCUSSION THREAD

**Discussion by cobbcheng** [aircodelabs/aircode#50] [Q&A] [answered]
目前在用一款serverless服务，但是不支持websocket连接，只能自己配置云服务器另外开发一个ws服务。如果serverless平台直接支持了ws连接服务那再好不过了。

**Comment by adcentury** [comment, marked as answer]
你好，我们目前暂时还没有支持 WS 的计划。当然，你也可以描述一下你的使用场景，帮助我们来进行产品判断。
